uint8 agentId
float64[3] homeLocation
float64 nominalHeading
